omim2obo/main.py

```diff
@@ -270,32 +278,45 @@ def omim2obo(use_cache: bool = False):
             if len(assocs) == 1:
                 predicate = MORBIDMAP_PHENOTYPE_MAPPING_KEY_PREDICATES[p_map_key]
 
+            evidence = Literal(f'Evidence: ({p_map_key}) {p_map_lab}')
+            
             # i. Add to MIM class
             # noinspection DuplicatedCode  b_and_b3_diff_results
             b = BNode()
             graph.add((b, RDF['type'], OWL['Restriction']))
             graph.add((b, OWL['onProperty'], predicate))
-            if p_mim:
-                graph.add((b, OWL['someValuesFrom'], OMIM[p_mim]))
+            graph.add((b, OWL['someValuesFrom'], OMIM[p_mim]))
             graph.add((OMIM[mim_number], RDFS['subClassOf'], b))
 
             # ii. add axiom
             b2 = BNode()
             graph.add((b2, RDF['type'], OWL['Axiom']))
             graph.add((b2, OWL['annotatedSource'], OMIM[mim_number]))
             graph.add((b2, OWL['annotatedProperty'], RDFS['subClassOf']))
-            # todo: add mapping key and related info in a different way?
-            evidence = Literal(f'Evidence: ({p_map_key}) {p_map_lab}')
-            # todo: add biolink:GeneDiseaseAssociation structure as well?
+            graph.add((b2, OWL['annotatedTarget'], b))
             graph.add((b2, BIOLINK['has_evidence'], evidence))
             graph.add((b2, RDFS['comment'], evidence))
-            # noinspection DuplicatedCode b_and_b3_diff_results
-            b3 = BNode()
-            graph.add((b3, RDF['type'], OWL['Restriction']))
-            graph.add((b3, OWL['onProperty'], predicate))
-            if p_mim:
-                graph.add((b3, OWL['someValuesFrom'], OMIM[p_mim]))
-            graph.add((b2, OWL['annotatedTarget'], b3))
+            
+            if predicate in MORBIDMAP_PHENOTYPE_MAPPING_KEY_INVERSE_PREDICATES:
+                # The following code basically just adds the converse relation:
+                # If there is g2d in the previous code, we now, in addition, add d2g
+                inverse_predicate = MORBIDMAP_PHENOTYPE_MAPPING_KEY_INVERSE_PREDICATES[predicate]
+               
+                # i. Add to MIM class
+                # noinspection DuplicatedCode  b_and_b3_diff_results
```

Comment:
I'm guessing you accidentally copied over this JetBrains `noinspection` comment that I went over in [this comment](https://github.com/monarch-initiative/omim/pull/107/files#r1477090456), along with my own personal comment `b_and_b3_diff_results`?

Reply:
Yep you are right, I removed this from here.